src/components/blocks/CallToAction/call-to-action-block.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { tinaField } from "tinacms/dist/react";
+import { ModalButton } from "../../ui/modalButton";
+import { Icon } from "../../icon";
+
+interface Action {
+  title?: string;
+  description?: string;
+  selectedIcon?: string;
+}
+
+interface CallToActionProps {
+  title?: string;
+  description?: string;
+  actions?: Action[];
+  buttonText?: string;
+}
+
+export const CallToActionBlock = ({ data }: { data: any }) => {
+  const { title, description, actions, buttonText }: CallToActionProps = data;
+  return (
+    <section className="py-16">
+      <div className="container mx-auto px-4">
+        {title && (
+          <div className="text-center mb-4">
+            <h2
+              className="text-5xl font-bold mb-10"
+              data-tina-field={tinaField(data, "title")}
+            >
+              {title}
+            </h2>
+            {description && (
+              <div
+                className="max-w-[512px] mx-auto text-lg text-[#B5B3AD] px-8"
+                data-tina-field={tinaField(data, "description")}
+              >
+                {description}
+              </div>
+            )}
+          </div>
+        )}
+
+        {actions && actions.length > 0 && (
+          <div className="flex flex-wrap mb-5 gap-16 text-center items-center justify-center max-w-[1124px] mx-auto px-4">
```

Comment:
same as above, this one doesnt have a direct correlation to tailwind but often i would choose closest and confirm with designer 
```suggestion
          <div className="flex flex-wrap mb-5 gap-16 text-center items-center justify-center max-w-6xl mx-auto px-4">
```

Reply:
done - correct value is 1024px so I added max-w-5xl